hydrocarbon_problem/api/testing_SvK.ipynb

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
Are the ipynb notebooks relevant or can they be removed? (If they do useful visualisation/testing that is not possible in `api_tests` then we can keep them).